source/image-handler/semantic-mapper.ts

```diff
@@ -14,13 +15,15 @@ export class SemanticMapper {
    * @returns Image edits based on the request path.
    */
   public mapPathToEdits(event: ImageHandlerEvent): ImageEdits {
+    this.originalFormat = event.path.substring(event.path.lastIndexOf(".") + 1);
 
-    if ([event.multiValueQueryStringParameters?.h,
-    event.multiValueQueryStringParameters?.w,
-    event.multiValueQueryStringParameters?.fit,
-    event.multiValueQueryStringParameters?.fm,
-    event.multiValueQueryStringParameters?.q]
-      .some((p) => { p?.length > 1; })) {
+    if (this.originalFormat.toLocaleLowerCase() === "svg") {
+      return this.EMPTY_IMAGE_EDITS;
+    }
```

Comment:
I would allow for `originalFormat` being `svg` and `fm` parameter specified to also continue.

If no fm is specified, the early exit is 👍